src/System.Net.Primitives/tests/FunctionalTests/CookieTest.cs

```diff
@@ -337,5 +337,26 @@ public static void ToString_Compare_Success()
             c.Port = "";
             Assert.Equal("name=value; $Path=path; $Domain=domain; $Port", c.ToString());
         }
+        
+        [Theory]
+        [MemberData(nameof(HashCodesTestData))]
+        public static void HashCodes_ShouldWorkLikeTuple(string name, string value, string path, string domain, int version)
```

Comment:
> "string".GetHashCode() will be different every time you run the app.

@jamesqo Thanks for explaining! In that case, I think it's fine to leave it the way it's written now.